src/plugins/elasticsearch/index.js

```diff
@@ -28,8 +28,8 @@ module.exports = function ({ Plugin }) {
           cert: string(),
           key: string()
         }).default(),
-        apiVersion: string().default('2.0'),
-        engineVersion: string().valid('^2.1.0').default('^2.1.0')
+        apiVersion: Joi.string().default('master'),
```

Comment:
Well, since the elasticsearch installation is pinned to master we need to use the same API version. The API in use in es master isn't shipping in any other version.